docs/content/en/api.md

```diff
@@ -106,24 +105,29 @@ The key name for the content-holding property
 
 The key name for possible meta templates
 
-### refreshOnceOnNavigation <Badge text="runtime" type="yellow" />
+### refreshOnceOnNavigation
+<badge>runtime</badge>
 - type `boolean`
 - default `false`
 
 When `true` then `vue-meta` will pause updates once page navigation starts and resumes updates when navigation finishes (resuming also triggers an update).
 This could both be a performance improvement as a possible fix for 'flickering' when you are e.g. replacing stylesheets
 
-:::tip
+<alert type="info">
+
 Its not supported to disable `refreshOnceOnNavigation` once enabled
-:::
 
-### debounceWait <Badge text="v2.3+"/><Badge text="runtime" type="yellow" />
+</alert>
+
+### debounceWait
+<badge>v2.3+</badge><badge>runtime</badge>
```

Comment:
Does nuxt content support different colors? I would prefer that as it helps with recognising